bridges/relays/lib-substrate-relay/src/messages/source.rs

```diff
@@ -63,6 +63,16 @@ use std::ops::RangeInclusive;
 pub type SubstrateMessagesProof<C, L> = (Weight, FromBridgedChainMessagesProof<HashOf<C>, L>);
 type MessagesToRefine<'a> = Vec<(MessagePayload, &'a mut OutboundMessageDetails)>;
 
+/// Outbound lane data - for backwards compatibility with `bp_messages::OutboundLaneData` which has
+/// additional `lane_state` attribute.
+#[derive(Decode)]
+struct OutboundLaneDataWrapper {
```

Comment:
Nit: I think a more apropriate name would be `LegacyOutboundLaneData`. The same for `InboundLaneData`.

Reply:
I am also not happy with `*Wrapper`, previously I had there `OutboundLaneWithoutState`, but I am also not sure about Legacy, these structs are just "subset" of Inbound/OutboundLaneData,

anyway, fixed: https://github.com/paritytech/polkadot-sdk/pull/5921/commits/9c91f79ce7b9b64f2e41894160fa6fbeda62e12a